### DISCUSSION TITLE
DFM17 as GPS for rdz_ttgo_sonde

### DISCUSSION THREAD

**Discussion by rpratt20** [dl9rdz/rdz_ttgo_sonde#427] [Q&A]
I have connected a DFM17 gps to my ttgo using the 3.3v, gnd, pin 13 as gps rx and pin 12 as gps tx.  The gps seems to lock and gives the 1 sec blinking led.  A click on the live map receiver icon shows sats: 0 and appears to be configured for gps location but I do not get any coordinates and the icon never moves.  I am looking for advice from someone who may have succeeded in this.  Also, I removed the STM32, radio, and battery holders from the DFM board.   The DFM board is powered from pin 1 on the swd pad at 3.3v, ground to the former battery negative, and gps tx rx to the larger pass through holes near pins 2, 3 of the ublox chip. I wonder if some other chip in the DFM is resetting the ublox to other than default configuration.
Randy

**Comment by dl9rdz** [comment]
Maybe start with some simple software on the TTGO that just dumps data from the GPS chip to the Serial port, so you can see if the GPS chip actually sends data in NMEA format.  If not try a factory reset of the GPS chip.

**Comment by rpratt20** [comment]
I found out the connection points do function as intended electrically but mechanically are extremely fragile. It becomes nearly impossible to not crack the solder joints within the pass through holes while making further connections.  The wires will appear connected nicely but the connection will be intermittent.  As a work around, I cleaned short lengths of the trace near the holes.  Then used fine #34 enamel wire passing from behind the board, through the hole and soldered flat to the trace.  Mechanically attach the fine wire to the back of the board with hot glue or tape.  Finally make the connection to the ESP board with other wire or your desired method.  While figuring this out I also did the GPS reset described in the Wiki but I don't know for certain if it would have been required because I had also connected to an Arduino Nano attempting to verify GPS output.
